src/seedu/addressbook/commands/CommandResult.java

```diff
@@ -20,6 +20,9 @@ public CommandResult(String feedbackToUser) {
         this.feedbackToUser = feedbackToUser;
         relevantPersons = null;
     }
+    public String GetCommandResult() {
+    	return feedbackToUser;
+    }
```

Comment:
Avoid accumulating unnecessary commits. I think I have told you this before as well. Please be more careful.